src/platform/desktop_notifications/desktopnotify.cpp

```diff
@@ -51,8 +52,16 @@ void DesktopNotify::notifyMessage(const NotificationData& notificationData)
         }
     }
 
-    // Fallback to QSystemTrayIcon.
-    if (d->icon) {
-        d->icon->showMessage(notificationData.title, notificationData.message, notificationData.pixmap);
+    if (!QSystemTrayIcon::supportsMessages()) {
+        qWarning() << "System does not support notifications.";
```

Comment:
I think most (all?) log prints in qTox are without terminating full stops?

Reply:
Done. If only we had a tokstyle to tell us about this.